webapp/src/features/adminEditIntersection/AdminEditIntersection.tsx

```diff
@@ -118,175 +129,252 @@ const AdminEditIntersection = () => {
 
   return (
     <Dialog open={open}>
-      <DialogTitle>Edit Intersection</DialogTitle>
       {Object.keys(apiData ?? {}).length != 0 ? (
         <>
-          <DialogContent>
+          <DialogContent sx={{ width: '600px', padding: '5px 10px' }}>
```

Comment:
issue: when editing I momentarily see the "unknown" message box before this correct edit form is shown